apis/objectstorage.k8s.io/v1alpha1/azure_types.go

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// +k8s:deepcopy-gen=false
+
+package v1alpha1
```

Comment:
Do we need protocol-specific types now? I may be a bit behind, but I thought the `Bucket` instance was no longer going to expose protocol-specific fields, other than, perhaps, common values for all protocols.

Reply:
@jeffvance This PR is inline with the KEP that is merged. We will iterate it when the new proposal is merged.